app/controllers/api/v1/search_results_controller.rb

```diff
@@ -8,26 +8,40 @@ class SearchResultsController < ApplicationController
 
       def create
         if upload_form.valid?
-          search_results = upload_form.save
+          search_result_list = upload_form.save
 
-          message = I18n.t('activemodel.notices.models.search_result.create')
-          search_results_serializer = SearchResultsSerializer.new(search_results, meta: { message: message })
+          upload_meta = upload_meta(search_result_list: search_result_list)
+          search_result_serializer = SearchResultSerializer.new(search_result_list, upload_meta).serializable_hash.to_json
```

Comment:
This can be omitted 🤔 
```suggestion
          search_result_serializer = SearchResultSerializer.new(search_result_list, upload_meta)
```